core/modules/node/node.install

```diff
@@ -1173,6 +1173,23 @@ function node_update_1016() {
   }
 }
 
+/**
+ * Update content admin view to show scheduled publication info.
+ */
+function node_update_1017() {
+  // See if the content admin view has been modified on this site.
+  $config = config('views.view.node_admin_content');
+
+  // Only update if view isn't overridden
+  if ($config->get('storage') != VIEWS_STORAGE_OVERRIDE) {
+    $path = backdrop_get_path('module', 'node') . '/config/';
+    $contents = file_get_contents($path . 'views.view.node_admin_content.json');
+    $data = json_decode($contents, true);
+    $config->setData($data);
+    $config->save();
```

Comment:
I don't think we can just pull in the default configuration like this. Currently we have to make these changes explicitly by duplicating the changed config into the update hook. See https://github.com/backdrop/backdrop-issues/issues/3347.

Reply:
@quicksketch I added a commit via github to convert json to php array, but now have a conflict. I haven't tried fixing another person's branch so got to take a careful look on how.